pkg/ccl/backupccl/restore_processor_planning.go

```diff
@@ -279,18 +279,8 @@ func distRestore(
 		return errors.Wrap(err, "making distSQL plan")
 	}
 
-	replanner, stopReplanner := sql.PhysicalPlanChangeChecker(ctx,
```

Comment:
do we want to remove PhysicalPlanCHangeChecker since there are no callers remaining?  Or, we'll revisit it at some point?